x-pack/test/alerting_api_integration/security_and_spaces/tests/actions/builtin_action_types/resilient.ts

```diff
@@ -0,0 +1,549 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import expect from '@kbn/expect';
+
+import { FtrProviderContext } from '../../../../common/ftr_provider_context';
+
+import {
+  getExternalServiceSimulatorPath,
+  ExternalServiceSimulator,
+} from '../../../../common/fixtures/plugins/actions_simulators/server/plugin';
+
+const mapping = [
+  {
+    source: 'title',
+    target: 'name',
+    actionType: 'overwrite',
+  },
+  {
+    source: 'description',
+    target: 'description',
+    actionType: 'overwrite',
+  },
+  {
+    source: 'comments',
+    target: 'comments',
+    actionType: 'append',
+  },
+];
+
+// eslint-disable-next-line import/no-default-export
+export default function jiraTest({ getService }: FtrProviderContext) {
+  const supertest = getService('supertest');
+  const esArchiver = getService('esArchiver');
+  const kibanaServer = getService('kibanaServer');
+
+  const mockResilient = {
+    config: {
+      apiUrl: 'www.jiraisinkibanaactions.com',
+      orgId: '201',
+      casesConfiguration: { mapping },
+    },
+    secrets: {
+      apiKeyId: 'key',
+      apiKeySecret: 'secret',
+    },
+    params: {
+      subAction: 'pushToService',
+      subActionParams: {
+        caseId: '123',
```

Comment:
I prefer to name it as `savedObjectId` instead of `caseId` (did renaming in my PR for ServiceNow), but it could be changed later, when generic versions of the IBM Resilient and Jira will be introduced

Reply:
You are right. We gonna change it either way to match your changes at https://github.com/elastic/kibana/pull/68464.